apps/dh/api-dh/source/DataHub.WebApi/Options/SubSystemBaseUrls.cs

```diff
@@ -12,33 +12,50 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-namespace Energinet.DataHub.WebApi;
+using System.ComponentModel.DataAnnotations;
 
-public class ApiClientSettings
+namespace Energinet.DataHub.WebApi.Options;
+
+public sealed record SubSystemBaseUrls
 {
+    public const string SectionName = "SubSystemBaseUrls";
+
+    [Required]
     public string MeteringPointBaseUrl { get; set; } = string.Empty;
 
+    [Required]
     public string MarketParticipantBaseUrl { get; set; } = string.Empty;
 
+    [Required]
     public string WholesaleBaseUrl { get; set; } = string.Empty;
 
+    [Required]
     public string WholesaleOrchestrationsBaseUrl { get; set; } = string.Empty;
 
+    [Required]
     public string WholesaleOrchestrationSettlementReportsBaseUrl { get; set; } = string.Empty;
```

Comment:
Maybe we remove WholesaleOrchestrationSettlementReportsBaseUrl  and WholesaleOrchestrationSettlementReportsLightBaseUrl?

Reply:
Lets do it in an other PR